yarn.lock

```diff
@@ -5820,7 +5828,7 @@ fast-url-parser@^1.1.3:
   dependencies:
     punycode "^1.3.2"
 
-faye-websocket@0.11.1, faye-websocket@>=0.6.0:
+faye-websocket@>=0.6.0:
```

Comment:
this is used by the dev dependency on `firebase-tools`, which in turn pulls in an old version of `firebase` (this repo)